public/views/codimd/body.ejs

```diff
@@ -15,27 +15,27 @@
                     <a id="permissionLabel" class="ui-permission-label text-uppercase" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
                     </a>
                     <ul class="dropdown-menu" aria-labelledby="permissionLabel">
-                        <li class="ui-permission-freely"<% if(!allowAnonymous && !allowAnonymousEdits) { %> style="display: none;"<% } %>><a><i class="fa fa-leaf fa-fw"></i> Freely - Anyone can edit</a></li>
-                        <li class="ui-permission-editable"><a><i class="fa fa-shield fa-fw"></i> Editable - Signed-in people can edit</a></li>
-                        <li class="ui-permission-limited"><a><i class="fa fa-id-card fa-fw"></i> Limited - Signed-in people can edit (forbid guests)</a></li>
-                        <li class="ui-permission-locked"><a><i class="fa fa-lock fa-fw"></i> Locked - Only owner can edit</a></li>
-                        <li class="ui-permission-protected"><a><i class="fa fa-umbrella fa-fw"></i> Protected - Only owner can edit (forbid guests)</a></li>
-                        <li class="ui-permission-private"><a><i class="fa fa-hand-stop-o fa-fw"></i> Private - Only owner can view &amp; edit</a></li>
+                        <li class="ui-permission-freely"<% if(!allowAnonymous && !allowAnonymousEdits) { %> style="display: none;"<% } %>><a><i class="fa fa-leaf fa-fw"></i> <%= __('Freely - Anyone can edit') %></a></li>
+                        <li class="ui-permission-editable"><a><i class="fa fa-shield fa-fw"></i> <%= __('Editable - Signed-in people can edit') %></a></li>
+                        <li class="ui-permission-limited"><a><i class="fa fa-id-card fa-fw"></i> <%= __('Limited - Signed-in people can edit (forbid guests)') %></a></li>
+                        <li class="ui-permission-locked"><a><i class="fa fa-lock fa-fw"></i> <%= __('Locked - Only owner can edit') %></a></li>
+                        <li class="ui-permission-protected"><a><i class="fa fa-umbrella fa-fw"></i> <%= __('Protected - Only owner can edit (forbid guests)') %></a></li>
+                        <li class="ui-permission-private"><a><i class="fa fa-hand-stop-o fa-fw"></i> <%= __('Private - Only owner can view and edit') %></a></li>
                         <li class="divider"></li>
-                        <li class="ui-delete-note"><a><i class="fa fa-trash-o fa-fw"></i> Delete this note</a></li>
+                        <li class="ui-delete-note"><a><i class="fa fa-trash-o fa-fw"></i> <%= __('Delete this note') %></a></li>
                     </ul>
                 </span>
                 <br>
                 <span class="ui-owner" style="display: none;">
                     &thinsp;<i class="ui-user-icon small" data-toggle="tooltip" data-placement="right"></i>
-                    &nbsp;<span class="text-uppercase">owned this note</span>
+                    &nbsp;<span class="text-uppercase"><%= __('owned this note') %></span>
```

Comment:
I think this is a typo. It should be "owns this note" instead of "owned".

Reply:
I don't know where this message is shown so I'm not sure. Fell free to fix.